ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/deprovision/cucushift-installer-rehearse-aws-ipi-ovn-fips-deprovision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-fips-deprovision
+  steps:
+  - chain: gather-network
+  - chain: gather-core-dump
```

Comment:
better to gather more resources here, like https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/byo-route53/deprovision/cucushift-installer-rehearse-aws-ipi-byo-route53-deprovision-chain.yaml
```
  - ref: gather-aws-console
  - chain: gather
```